app/models/reimbursement/expense.rb

```diff
@@ -117,6 +117,13 @@ class Expense < ApplicationRecord
       end
     end
 
+    before_update do
+      if approved? && (memo_changed? || amount_cents_changed? || category_changed? || description_changed?)
```

Comment:
imo we should only do this if the amount changes

Reply:
I don't have a super strong opinion on this, but maybe we should also check if the receipts changed?

Reply:
I'm fine with making it just the amount, my thought was that any change being made and keeping it approved could be misleading, but amount is definitely the most important. I considered checking receipts as well, but I didn't see a good way of doing it since the expense modal isn't actually updated when a receipt is uploaded, it's just a polymorphic relationship on receipt to any receiptable.

Reply:
Ok - let's run with this and iterate on it production